### ISSUE TITLE
Event.target is not assigned

### ISSUE THREAD

**Issue by mathew-kurian** [jpillora/xhook#77]
```
xhr.addEventListener('load', event => event.target !== xhr);
```

**Comment by morsdyce** [comment]
Hi @bluejamesbond 

Looking at the spec there is no reference to the target have to be the same XHR object. In addition there is no currently trivial way to create an event with the target specified.
I know this is a bit late, but can you tell me what the use case was when you submitted the issue?

Thanks

**Comment by banyudu** [comment]
Websites may use `event.target.status` or other attributes.

For example, [Docsify](https://github.com/docsifyjs/docsify/blob/0375f78fcd58677b729dc07eff8828444b52e4f3/src/core/util/ajax.js#L63)

```js
xhr.addEventListener('load', event => {
  const target = /** @type {XMLHttpRequest} */ (event.target);
  if (target.status >= 400) {
    error(event);
  }
}
```

**Comment by TrickyPi** [comment]
Closed by #166
